static_report/src/components/ComparisonReport/CRTableList/CRTableListAssertions.tsx

```diff
@@ -168,9 +181,12 @@ export function CRTargetTableAssertionsDifference({
         />
       ) : null}
       <Text as="span" color={isGreaterThanZero ? 'black' : 'inherit'}>
-        {difference === 0 && targetAssertions}
-        {difference < 0 && targetAssertions}
-        {difference > 0 && targetAssertions}
+        {difference === 0 &&
+          formatColumnValueWith(targetAssertions, formatNumber)}
+        {difference < 0 &&
+          formatColumnValueWith(targetAssertions, formatNumber)}
+        {difference > 0 &&
+          formatColumnValueWith(targetAssertions, formatNumber)}
```

Comment:
why are the render conditions necessary if the arguments are the same for `formatColumnValueWidth`? 🤔

Reply:
My fault, when i developed here and i list all conditions and I forgot to delete xd.